package.json

```diff
@@ -48,27 +29,12 @@
         "eslint-plugin-react-hooks": "^4.2.0",
         "eslint-plugin-simple-import-sort": "^5.0.3",
         "eslint-plugin-unused-imports": "^1.0.0",
-        "exceljs": "^3.4.0",
-        "file-saver": "^2.0.2",
-        "final-form": "^4.16.1",
-        "graphql": "^15.4.0",
-        "graphql-tag": "^2.11.0",
-        "history": "^4.10.1",
         "husky": "^4.3.8",
-        "jss": "^9.8.7",
-        "lerna": "^3.15.0",
+        "lerna": "^4.0.0",
         "lint-staged": "^10.5.3",
         "prettier": "^2.1.2",
-        "react": "^16.8.6",
-        "react-dom": "^16.8.6",
-        "react-final-form": "^6.3.1",
-        "react-intl": "^5.10.0",
-        "react-router": "^5.1.2",
-        "react-router-dom": "^5.1.2",
-        "react-select": "^3.0.4",
         "rimraf": "^2.6.3",
-        "styled-components": "^4.3.2",
```

Comment:
is removing those deps possible because npm 7 now installs peerDepdences?

Reply:
yes, because npm 7 auto installs peer dependencies we can cleanup the root package.json